### ISSUE TITLE
Show a different message when Chrome App installed but the port is closed

### ISSUE THREAD

**Issue by sirdarckcat** [google/tamperchrome#21]
See #20 for more details

**Comment by sirdarckcat** [comment]
This will also be obsolete when #1 is fixed, but it's worth doing first, since it's a small change.

**Comment by sirdarckcat** [comment]
@eliyastein since you made some changes recently here, are you interested on this? it's a small patch and it is similar to your last change, except here we need to check the response on https://github.com/google/tamperchrome/blob/187a02c2cb4fa4e624062a9b4cd907ee5ec5a6e4/extension/background.js#L35 to confirm the extension is installed before showing the message here https://github.com/google/tamperchrome/blob/187a02c2cb4fa4e624062a9b4cd907ee5ec5a6e4/extension/tamper.js#L66

in other words, before showing appNotFound() we need to check if background.js' message to the background app actually failed (or not).

https://developer.chrome.com/extensions/messaging#external

**Comment by eliyastein** [comment]
@sirdarckcat - Happy to take a look, but might be a week or so before I can get to it.

**Comment by sirdarckcat** [comment]
thanks! that works for me :)

**Comment by sirdarckcat** [comment]
#18 and #31 obsolete the chrome app completely, so closing this
